src/components/DotNavigation/docs/index.story.tsx

```diff
@@ -56,30 +62,48 @@ export default {
           title('Examples'),
 
           ...[
-            { title: 'Default Example', source: examples.example },
-            { title: 'Three Dot Example', source: examples.threeDotExamle },
+            {
+              title: 'Default Example',
+              description: 'By default the DotNavigation shows 5 dots',
+              source: examples.example,
+            },
+            {
+              title: 'Three Dot Example',
+              description: 'The number of dots can be less than 5 though',
+              source: examples.threeDotExamle,
+            },
             {
               title: 'Ten Dot Example',
+              description:
+                'When there are more than 5 items, the DotNavigation shows a maximum of 5 dots, but with an indication that there are more to navigate to',
               source: examples.tenDotExamle,
             },
-            { title: 'Light Example', source: examples.lightExample },
+            {
+              title: 'Light Example',
+              description: 'The DotNavigation has a light theme',
+              source: examples.lightExample,
+            },
             {
               title: 'Light Example with Safety Border',
+              description:
+                'A "safety border" can be added to the dots, in case the background has a similar color as the dots',
               source: examples.lightExampleWithSafetyBorder,
             },
             {
               title: 'Default Example with currentIndex = 2',
+              description:
+                'The current selected index can be configured by a prop',
               source: examples.defaultExampleWithCurrentIndex,
             },
-          ].map(code),
+          ].map(example),
         ],
       }),
 
       ...[
+        { title: 'Playground', sections: [playground()] },
```

Comment:
```suggestion
        { title: 'Playground', sections: [playground(), autoSettingsPanel()] },
```
?

Reply:
I prefer not to add at the moment.
Not until we make it work